gn3/api/correlation.py

```diff
@@ -54,10 +76,10 @@ def compute_tissue_corr(corr_method="pearson"):
     """api endpoint fr doing tissue correlation"""
     tissue_input_data = request.get_json()
     primary_tissue_dict = tissue_input_data["primary_tissue"]
-    target_tissues_dict_list = tissue_input_data["target_tissues"]
+    target_tissues_dict = tissue_input_data["target_tissues_dict"]
```

Comment:
*question*: What happens when "target_tissues_dict" is absent? Normally, when fetching things from a dict, I prefer using "get", since you can give it has a default value of None when a key is absent. In the former case, an error is thrown which I think closes the flask server in certain cases(for example if you assumed that you had a value and tried to run something using said value).

Cc @zsloan

Reply:
Will add a check for None results.